### ISSUE TITLE
Game design: how many walls make a room?

### ISSUE THREAD

**Issue by VynceMontgomery** [VynceMontgomery/sankai#18]
It is perhaps unaesthetic that a room is 3 walls. so consider: 
1 fortress = 3 towers = 9 buildings = 27 rooms = 81 walls = 243 lumber
1 fortress = 4 towers = 16 rooms = 64 walls = 256 lumber

-1 level, +1 basis factor... nearly the same total cost. and more aesthetic. 

This would definitely require changing the die; even L W R A C * might be too fast. (though L L W R C * seems slow, and L L W A C * seems _really_ slow)

Removing a level also shortens Peko's hunger track, but if we make each level there 5 long, it's about the same length it was before. (16 vs 17). Make it 6 and it's 19, which might be better since we are (a) lengthening the builder track a fraction and (b) peko was usually winning aside from shenanigans anyway. 

In short, from a game design aspect this may be an improvement, too - wider levels reduces cheesing, the die rolling Harapeko's Appetite makes i harder to stalemate, as the builder is forced (eventually)to build what peko wants.
